README.md

```diff
@@ -120,3 +135,295 @@ This will return the following runtime images with the following names: `action-
 ```
 ./gradlew :tests:test
 ```
+
+---
+
+## Using Docker <a name="using_docker"></a>
+
+### Pre-requisites
+- [Docker](https://www.docker.com/)
+- [curl](https://curl.se/), [wget](https://www.gnu.org/software/wget/), or [Postman](https://www.postman.com/)
```

Comment:
add some reason why you would choose this (manual) method over GRadle... e.g., 
"The Gradle Wrapper build method uses Docker internally to automate many functions.  If you prefer building and testing with Docker directly, we have provided the following instructions"